### PR TITLE
meta tweaks

### PR DESCRIPTION
**Pull request by SamVerschueren**
- Add `*.yml` to `.editorconfig`
- Added XO
- Run tests on Node `5` and `4` instead of `iojs`
- Tweaked documentation
- Split up tests so it's easier to detect if one fails

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -34,11 +34,13 @@ emailRegex({exact: true}).test('sindresorhus@gmail.com');
...
+### emailRegex([options])
 
 Returns a regex for matching email addresses.
 
-#### options.exact
+#### options
+
+##### exact
 
 Type: `boolean`
```
Comment (sindresorhus) [inline readme.md]: Add `<br>` to the end. I've started to use that instead of whitespace.

File: readme.md [hunk 2]
```diff
@@ -34,11 +34,13 @@ emailRegex({exact: true}).test('sindresorhus@gmail.com');
...
 
 Returns a regex for matching email addresses.
 
-#### options.exact
+#### options
+
+##### exact
 
 Type: `boolean`  
 Default: `false` *(Matches any email address in a string)*
```
Comment (sindresorhus) [inline readme.md:46]: Change the bottom URL from `http` to `https`.

File: readme.md (hunk 1 above)
Comment (sindresorhus) [inline readme.md]: And remove line 13-15 in .editorconfig.

**Comment by sindresorhus** [comment]
And remove `http://` here: https://github.com/sindresorhus/email-regex/blob/master/package.json#L10

**Comment by SamVerschueren** [comment]
done :+1:

**Comment by sindresorhus** [comment]
Oh, and add XO: `$ xo --init --unicorn`.

File: readme.md [hunk 3]
```diff
@@ -34,19 +34,21 @@ emailRegex({exact: true}).test('sindresorhus@gmail.com');
...
+#### options
 
-Type: `boolean`  
+##### exact
+
+Type: `boolean`<br>
 Default: `false` *(Matches any email address in a string)*
 
-Only match an exact string.  
+Only match an exact string.
```
Comment (sindresorhus) [inline readme.md]: `<br>`

**Comment by SamVerschueren** [comment]
XO was already added.

**Comment by sindresorhus** [comment]
Thanks for the cleanup :)

**Comment by SamVerschueren** [comment]
Always happy to help ;)

**Pull request closed without merging**